src/content/structured/accessibility/coding/semantic-html.mdx

```diff
@@ -16,9 +16,9 @@ contribute: "https://github.com/mi6/ic-design-system/tree/main/src/content/struc
 
 Use semantic HTML elements.
 
-Most elements are semantic such as `<form>` and `<table>` which clearly define its content. However, non-semantic elements such as `<div>` and `<span>` are often overused. These don't tell us about the meaning of the content.
+Semantic elements, such as `<form>` and `<table>`, clearly define their content. However, non-semantic elements, such as `<div>` and `<span>`, are often overused. These don't tell us about the meaning of the content.
```

Comment:
The new structure of this sentence reads a lot better to me, but it also flows fine without any commas in it - currently the two commas break it up a bit too much imo

Reply:
I agree!

Reply:
Can we keep it with the commas? The commas surround a clause that could removed without impacting whether the sentence still makes sense, so commas are needed. See here for an example explanation: https://www.elevenwriting.com/blog/comma-before-such-as

Reply:
I vote for removing the however - and you could join the next sentence in as a clause?
> non-semantic elements, such as `<div>` and `<span>`, are often overused, **they** don't tell us about the meaning of **their** content.

(changes in **bold**)